introduction.md

```diff
@@ -0,0 +1 @@
+Hello! my name is Tania and this is my first time coming to this dojo. Namaste
```

Comment:
Hi codeschool-kiddo! I have made the necessary changes and pushed them up to my forked repo!